pkg/conference/config.go

```diff
@@ -2,7 +2,11 @@ package conference
 
 // Configuration for the group conferences (calls).
 type Config struct {
-	// Keep-alive timeout for WebRTC connections. If no keep-alive has been received
-	// from the client for this duration, the connection is considered dead (in seconds).
+	// Keep-alive timeout for WebRTC connections. If the client doesn't respond
+	// to an `m.call.ping` with an `m.call.pong` for this amount of time, the
+	// connection is considered dead. (in seconds, no greater then 30)
 	KeepAliveTimeout int `yaml:"timeout"`
+	// The time after which we should send another m.call.ping event to the
+	// client. (in seconds, greater then 30)
+	PingInterval int `yaml:"pingInterval"`
```

Comment:
Making this configurable is probably overkill and will give people another way to shoot themselves in the foot, but we can always comment it as, "you really don't want to change this" in the config.

Reply:
Well, we validate the config, so I think we don't have to worry about it much?

Reply:
True... Will add some warning too when adding the config errors

Reply:
(https://github.com/matrix-org/waterfall/issues/71)